src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -4,107 +4,75 @@ short-title: Supported platforms
 
-<div class="table-wrapper" markdown="1">
-|Platform|Version                       |Channels |
-|--------|------------------------------|---------|
-|Android | API 16 (Android 4.1) & above | All     |
-|iOS     | iOS 11 & above               | All     |
-|Linux   | Debian, 64-bit               | All     |
-|macOS   | Mojave (10.14) & above       | All     |
-|Web     | Chrome 84  & above           | All     |
-|Web     | Firefox 72.0 & above         | All     |
-|Web     | Safari on El Capitan & above | All     |
-|Web     | Edge 1.2.0 & above           | All     |
-|Windows | Windows 7 & above, 64-bit    | All     |
-{:.table.table-striped}
-</div>
+## Deploying Flutter
 
-All channels include master, beta,
-and stable channels.
-
-### Google-tested platforms
+As of the current stable release,
+support for deploying Flutter apps is shown in the
+following table:
 
 <div class="table-wrapper" markdown="1">
-|Platform|Version               |
-|--------|----------------------|
-|Android |Android SDK 19–30*    |
-|iOS     |14-15                 |
-|Linux   |Debian 10             |
-|Linux   |Ubuntu 18.04 LTS      |
-|macOS   |Monterey (12) & above |
-|Web     |Chrome 84             |
-|Web     |Firefox 72.0          |
-|Web     |Safari / Catalina     |
-|Web     |Edge 1.2.0            |
-|Windows |Windows 10            |
+|Platform version        | Supported | Best effort | Unsupported |
+|------------------------|-----------|-------------|-------------|
+| Android SDK 19-30<sup>1</sup> | ✓  |             |             |
+| Android SDK 16-18      |           |      ✓      |             |
+| Android SDK 15 & below |           |             |      ✓      |
+| iOS 14-15              |     ✓     |             |             |
+| iOS 11-13              |           |      ✓      |             |
+| [iOS 10 & below & `arm7v` 32-bit iOS][] |        |      ✓      |
+| Linux Debian 11        |           |      ✓      |             |
+| Linux Debian 10        |     ✓     |             |             |
+| Linux Debian 9 & below |           |      ✓      |             |
+| Linux Ubuntu 22.04 |(aspirational) |      ✓      |             |
+| Linux Ubuntu 20.04     |           |      ✓      |             |
+| Linux Ubuntu 18.04 LTS |     ✓     |             |             |
+| Linux (any 32-bit platform) |      |             |      ✓      |
+| macOS Monterey (12) & above |  ✓   |             |             |
+| macOS Mojave (10.14) - Big Sur (11) | |   ✓      |             |
+| macOS High Sierra (10.13) & below | |            |      ✓      |
+| web - Chrome 84        |     ✓     |             |             |
+| web - Firefox 72.0     |     ✓     |             |             |
+| web - Safari/Catalina  |     ✓     |             |             |
+| web - Edge 1.2.0       |     ✓     |             |             |
+| Windows 10             |     ✓     |             |             |
+| Windows 11          |(aspirational)|      ✓      |             |
```

Comment:
Can we order consistently from newest to oldest? (Win10 and 11 should be flipped)...